samples/sysbuild/with_mcuboot/sample.yaml

```diff
@@ -7,6 +7,35 @@ tests:
     # Platform allowed is used as twister using sysbuild still lacks proper
     # filtering support, see discussion in #49552.
     platform_allow:
+      - frdm_k22f
```

Comment:
what is the purpose of adding all those platforms into the sample? This is not a test, the same demonstrates sysbuild, it is not a test to verify platforms. 
How is this going to end if each vendor added their platforms in here?

Reply:
see https://docs.zephyrproject.org/latest/samples/sample_definition_and_criteria.html#definition-and-criteria

Reply:
Hi @nashif
It's clear, but in this case, the mcuboot is not supported by all platforms. The example already has other allowed platforms so Twister ignores  platforms if they are not allowed.

Reply:
> what is the purpose of adding all those platforms into the sample? This is not a test, the same demonstrates sysbuild, it is not a test to verify platforms. How is this going to end if each vendor added their platforms in here?

Hi @nashif, 
All  Zephyr MCU-boot dependent samples/tests use the platform_allow list.
Could you unblock the PR?